Code/Source/ROS2SystemComponent.h

```diff
@@ -65,6 +66,7 @@ namespace ROS2
     private:
         std::shared_ptr<rclcpp::Node> m_ros2Node;
         AZStd::shared_ptr<rclcpp::executors::SingleThreadedExecutor> m_executor;
+        AZStd::unique_ptr<tf2_ros::TransformBroadcaster> m_dynamicTFBroadcaster;
         AZStd::unique_ptr<tf2_ros::StaticTransformBroadcaster> m_staticTFBroadcaster;
```

Comment:
You could technically make an [AZStd::variant](https://en.cppreference.com/w/cpp/utility/variant) that stores both a TransformBroadcaster and a StaticTransformBroadcaster to replacate a type safe union.
i.e
```
using TransformBroadcaster = AZStd::varaiant<AZStd::unique_ptr<tf2_ros::StaticTransformBroadcaster>, AZStd::unique_ptr<tf2_ros::TransformBroadcaster>>;
TransformBroadcaster m_staticTFBroadcaster;
```

Reply:
Both static and dynamic broadcaster are always needed (each robot will typically have both static and dynamic frame transforms).

Reply:
Ok thanks for the info.